prow/cmd/tide/README.md

```diff
@@ -1,180 +1,30 @@
-be passing to merge a PR.
-
-When branch protection is not used, required and optional contexts can be defined
-globally, or at the org, repo or branch level.
-
-If we want to skip unknown checks (ie checks that are not defined in Prow Config), we can set
-`skip-unknown-contexts` to true. This option can be set globally or per org,
-repo and branch.
-
-**Important**: If this option is not set and no prow jobs are defined tide will trust the github
-combined status and will assume that all checks are required (except tide).
-
-
-### Example
-
-```yaml
-tide:
-  merge_method:
-    kubeflow/community: squash
-
-  target_url: https://prow.k8s.io/tide.html
-
-  queries:
-  - repos:
-    - kubeflow/community
-    - kubeflow/examples
-    labels:
-    - lgtm
-    - approved
-    missingLabels:
-    - do-not-merge
-    - do-not-merge/hold
-    - do-not-merge/work-in-progress
-    - needs-ok-to-test
-    - needs-rebase
-
-  context_options:
-    # Use branch protection options to define required and optional contexts
-    from-branch-protection: true
-    # Treat unknown contexts as optional
-    skip-unknown-contexts: true
-    orgs:
-      org:
-        required-contexts:
-        - "check-required-for-all-repos"
-        repos:
-	  repo:
-            required-contexts:
-             - "check-required-for-all-branches"
-            branches:
-              branch:
-                from-branch-protection: false
-                required-contexts:
-                - "required_test"
-                optional-contexts:
-                - "optional_test"
-
-```
-
-**Explanation**: The component starts periodically querying all PRs in `github.com/kubeflow/community` and
-`github.com/kubeflow/examples` repositories that have `lgtm` and `approved` labels set
-and do not have `do-not-merge`, `do-not-merge/hold`, `do-not-merge/work-in-progress`, `needs-ok-to-test` and `needs-rebase` labels set.
-All PRs that conform to the criteria are processed and merged.
-The processing itself can include running jobs (e.g. tests) to verify the PRs are good to go.
-At the same time all commits in PRs from `github.com/kubeflow/community` repository are squashed before merging.
-
-## Presubmits
-
-Before a PR is merged, Tide ensures that all jobs configured as required (with `skip_report: false`) in the `presubmits` part of the `config.yaml` file
-are passing against the latest base branch commit, rerunning the jobs if necessary.
-
-Semantic of individual fields of the `presubmits` is described in [prow/README.md#how-to-add-new-jobs](https://github.com/kubernetes/test-infra/blob/master/prow/README.md#how-to-add-new-jobs).
+Tide was created in 2017 by @spxtr to replace [Mungegithub's](/mungegithub) Submit Queue.  It was designed to manage a large number of repositories across organizations without using many API rate limit tokens by identifying mergeable PRs with GitHub search queries fulfilled by GitHub's v4 GraphQL API.
```

Comment:
so selfless :)

Reply:
Joe deserves the credit. If he hadn't come up with Tide I might have written something scary to manage a fleet of mungegithub instances. :nauseated_face: